src_common/common_namespace_declarations.H

```diff
@@ -50,7 +50,8 @@ AMREX_GPU_MANAGED amrex::GpuArray<amrex::Real, MAX_SPECIES> common::molmass;
 AMREX_GPU_MANAGED amrex::GpuArray<amrex::Real, MAX_SPECIES> common::rhobar;
 AMREX_GPU_MANAGED amrex::Real common::rho0;
 AMREX_GPU_MANAGED amrex::GpuArray<amrex::Real, MAX_SPECIES> common::diameter;
-amrex::Vector<int>         common::dof;
+AMREX_GPU_MANAGED amrex::GpuArray<amrex::Real, MAX_SPECIES> common::dof;
```

Comment:
```suggestion
AMREX_GPU_MANAGED amrex::GpuArray<int, MAX_SPECIES> common::dof;
```

Reply:
If dof is changed to an int, the changes will need to propagate to all function signatures as well.